core/src/main/java/org/pegdown/ParserWithDirectives.java

```diff
@@ -251,7 +252,11 @@ public Rule DirectiveSource() {
     }
 
     public Rule DirectDirectiveSource() {
-        return Sequence(Sp(), Enclosed('(', ANY, ')'), push(new DirectiveNode.Source.Direct(popAsString())));
+        return Sequence(
+                Sp(),
+                Enclosed('(', DirectiveLitChar(), ')'),
+                push(new DirectiveNode.Source.Direct(popAsString().replaceAll("\\\\(.)", "$1")))
+        );
```

Comment:
Not sure if this is the best way to fix this, this is to allow you to put paranthesis in URLs, eg:

```
@javadoc[example](com.Example#example\(\))
```

I wasn't really in the mood to try and truly understand parboiled, this worked so I used it.